foursight_core/react/api/cognito.py

```diff
@@ -216,7 +231,7 @@ def _get_cognito_oauth_token_endpoint_authorization() -> str:
     :returns: Authorization header value for Cognito /oauth2/token endpoint.
     """
     config = _get_cognito_oauth_config_basic()
-    client_id = config["client_id"]
+    client_id = config["client"]
```

Comment:
Again, possibly object-oriented, but even if not, I think it's a good idea to put the keys into global variables. That gets you completion when editing and avoids misspelling.

Reply:
Sounds good, created constants for these.